unit/java_bytecode/java_bytecode_parse_generics/parse_bounded_generic_class.cpp

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************\
+
+ Module: Unit tests for parsing generic classes
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <testing-utils/catch.hpp>
+#include <testing-utils/load_java_class.h>
+#include <iostream>
+#include <istream>
+#include <memory>
+
+#include <util/config.h>
+#include <util/language.h>
+#include <util/message.h>
+#include <java_bytecode/java_bytecode_language.h>
+#include <java_bytecode/java_types.h>
+
+SCENARIO(
+  "java_bytecode_parse_generic_class_bound",
```

Comment:
Can you please change both the scenario name and the file name to `parse_bounded_generic_classes`?

Reply:
done